styles.css

```diff
@@ -59,6 +59,51 @@ a {
   display: none;
 }
 
+.visible {
+  display: flex;
```

Comment:
- [ ] It would be better if we avoid repeating CSS rules, like for example `display: flex` has appeared more than once so we can fix it by creating a class for `display: flex` as `.flex { display: flex }` and pass the class to all elements with the display property